Superbuild/External_GLEW.cmake

```diff
@@ -0,0 +1,54 @@
+############
+# This code was adapted from https://github.com/Slicer/Slicer/blob/main/SuperBuild/External_zlib.cmake
+############
+set(proj GLEW)
+
+set(${proj}_DEPENDENCIES "")
+
+if(Autoscoper_USE_SYSTEM_${proj})
+	unset(GLEW_ROOT CACHE)
+	find_package(GLEW REQUIRED)
+	set(GLEW_INCLUDE_DIR ${LIBGLEW_INCLUDE_DIR})
+	set(GLEW_LIBRARY ${LIBGLEW_LIBRARIES})
+endif()
+
+if(DEFINED GLEW_ROOT AND NOT EXISTS ${GLEW_ROOT})
+	message(FATAL_ERROR "GLEW_ROOT variable is defined but corresponds to nonexistent directory")
+endif()
+
+if(NOT DEFINED GLEW_ROOT AND NOT Autoscoper_USE_SYSTEM_${proj})
+	
+	set(EP_SOURCE_DIR ${CMAKE_BINARY_DIR}/${proj})
+	set(EP_BINARY_DIR ${CMAKE_BINARY_DIR}/${proj}-build)
+	set(EP_INSTALL_DIR ${CMAKE_INSTALL_PREFIX}/external)
+	
+	ExternalProject_Add(${proj}
+		URL https://sourceforge.net/projects/glew/files/glew/2.2.0/glew-2.2.0.zip
+		SOURCE_DIR ${EP_SOURCE_DIR}
+		BINARY_DIR ${EP_BINARY_DIR}
+		INSTALL_DIR ${EP_INSTALL_DIR}
+		CMAKE_ARGS
+			-S ${EP_SOURCE_DIR}/build/cmake/
+			-B ${EP_BINARY_DIR}
+			-DCMAKE_CXX_COMPILER=${CMAKE_CXX_COMPILER}
+			-DCMAKE_CXX_EXTENSIONS=${CMAKE_CXX_EXTENSIONS}
+			-DCMAKE_CXX_STANDARD_REQUIRED=${CMAKE_CXX_STANDARD_REQUIRED}
+		CMAKE_CACHE_ARGS
+			## CXX should not be needed, but it a cmake default test
```

Comment:
Example of comment to use for the option, see https://github.com/Slicer/Slicer/blob/main/Utilities/Templates/Extensions/SuperBuild/SuperBuild/External_Foo.cmake